inventory/routes/concepts/pull_push_rules.rst

```diff
@@ -0,0 +1,116 @@
+===========================
+What are Pull & Push Rules?
+===========================
+
+In inventory management, the supply chain strategy determines when
+products should be fabricated, delivered to distribution centers and
+made available in the retail channel.
+
+Under a pull supply chain, actual customer demand drives the process,
+while push strategies are driven by long-term projections.
+
+Pull Inventory Control System
+=============================
+
+The *procurement inventory control system* begins with a customer's
+order. With this strategy, companies only produce enough to fulfill their
+customer's orders. One advantage is that there are no
+extra goods that need to be stored, thus reducing inventory levels
+and the cost of carrying and storing. However, one major
+disadvantage to the pull system is that it is highly possible to run
+into ordering dilemmas, such as a supplier not being able to get a
+shipment out on time. This leaves the company unable to fulfill the
+order and contributes to customer dissatisfaction.
+
+An example of a pull inventory control system is the make-to-order. The
+goal is to keep inventory levels to a minimum by only having enough
+inventory to meet customer's demand. The :abbr:`MTO (Make-to-order)` system
+eliminates waste by reducing the amount of storage space needed for
+inventory and the costs of storing goods.
+
+Push Inventory Control System
+=============================
+
+The *push system of inventory control* involves forecasting inventory
+needs to meet customer's demand. Companies must predict which products
```

Comment:
I would add how they will predict, past sales, predicted increase due to various reasons etc